clarity/enhancer/gha/gha_interface.py

```diff
@@ -64,7 +64,7 @@ def create_configured_cfgfile(
             cfg_filename (str): cfg filename
         """
 
-        if self.fs != 44100:
+        if self.sample_frequency != 44100:
```

Comment:
This is a name we need to agree with. I always use `sample_rate`, I can see you use `sample_frequency`. I am happy with either.  I can re-name later mine with sample_frequency if you prefer.